src/ResourceManager/Resources/Commands.ResourceManager/Cmdlets/Implementation/Deployments/StopAzureDeploymentCmdlet.cs

```diff
@@ -0,0 +1,74 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System.Management.Automation;
+using Microsoft.Azure.Commands.ResourceManager.Cmdlets.Components;
+using Microsoft.Azure.Commands.ResourceManager.Cmdlets.SdkModels;
+using ProjectResources = Microsoft.Azure.Commands.ResourceManager.Cmdlets.Properties.Resources;
+
+namespace Microsoft.Azure.Commands.ResourceManager.Cmdlets.Implementation
+{
+    /// <summary>
+    /// Cancel a running deployment.
+    /// </summary>
+    [Cmdlet(VerbsLifecycle.Stop, "AzureRmDeployment", SupportsShouldProcess = true,
```

Comment:
@Tiano2017 this is probably called out in the design review, but the corresponding `Start-*` cmdlet is `New-AzureRmDeployment`, correct?

Reply:
Right. We talked about that. I tend to keep this because currently we have New-AzureRmResourceGroupDeployment. I suppose we should keep the same verb.

Reply:
@Tiano2017 got it. Maybe we should think about adding an alias for `New-AzureRmDeployment` that is `Start-AzureRmDeployment`? Something to think about